planner/core/preprocess_test.go

```diff
@@ -376,47 +376,5 @@ func (s *testValidatorSuite) TestDropGlobalTempTable(c *C) {
 	s.runSQL(c, "drop global temporary table temp, temp1", false, nil)
 	s.runSQL(c, "drop global temporary table temp, tb", false, core.ErrDropTableOnTemporaryTable)
 	s.runSQL(c, "drop global temporary table temp, ltemp1", false, core.ErrDropTableOnTemporaryTable)
-	s.runSQL(c, "drop temporary table xxxx, ltemp1", false, infoschema.ErrTableDropExists)
 	s.runSQL(c, "drop global temporary table test2.temp2, temp1", false, nil)
 }
-
-func (s *testValidatorSuite) TestDropLocalTempTable(c *C) {
```

Comment:
Well, the old code style is actually more reader friendly...

Reply:
Never mind ... just the coding style and inaccurate comment.
You are doing well enough, good job! @lcwangchao